mne/viz/backends/renderer.py

```diff
@@ -60,49 +60,46 @@ def set_3d_backend(backend_name, verbose=None):
+       +======================================+========+=========+==========+
+       | :func:`plot_vector_source_estimates` | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`plot_source_estimates`        | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`plot_alignment`               | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`plot_sparse_source_estimates` | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`plot_evoked_field`            | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`plot_sensors_connectivity`    | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`snapshot_brain_montage`       | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | :func:`link_brains`                  |        | ✓       |          |
+       +--------------------------------------+--------+---------+----------+
+       +--------------------------------------+--------+---------+----------+
+       | **Feature:**                                                       |
+       +--------------------------------------+--------+---------+----------+
+       | Large data                           | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Opacity/transparency                 | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Support geometric glyph              | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Smooth shading                       | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Subplotting                          | ✓      | ✓       | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Inline plot in Jupyter Notebook      | ✓      |         | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Inline plot in JupyterLab            | ✓      |         | ✓        |
+       +--------------------------------------+--------+---------+----------+
+       | Inline plot in Google Colab          |        |         |          |
+       +--------------------------------------+--------+---------+----------+
+       | Toolbar with jupyter widgets         |        |         | ✓        |
```

Comment:
I would just write "Toolbar" since users don't care what the underlying mechanism is
```suggestion
       | Toolbar                              |        | ✓       | ✓        |
```